skdata/mnist/dataset.py

```diff
@@ -202,8 +202,9 @@ def build_meta(self):
         assert len(arrays['test_images']) == len(arrays['test_labels'])
```

Comment:
Please chanage this also in fetch function.
`downloader = urllib.urlopen(url)`
`downloader = urllib.request.urlopen(url)`